.gitignore

```diff
@@ -34,11 +34,8 @@ Pods
 Carthage/
 
 # SPM
-Package.resolved
 .swiftpm
 .build
-*/xcshareddata/swiftpm
-SampleApps/SPMTest/SPMTest.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved
```

Comment:
I think we can keep this one in the gitignore

Reply:
@jaxdesmarais We removed this because it was not getting updated using "swift package resolve" command right?

Reply:
I think we want `*/xcshareddata/swiftpm` but @scannillo is right it looks like we have `SampleApps/SPMTest/SPMTest.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved` included in the BT `.gitignore`. We should revert the removal of `SampleApps/SPMTest/SPMTest.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved`.

Reply:
I reverted deletion of SampleApps/../Package.resolved in gitIgnore file in the commit  => 99c10102b0d453532d041b140ff717f9a09e9720

Reply:
Awesome!